ocean-gke-rootVolumeType-elastigroup-aws-resourceTagSpecification (#216)
spotinst/terraform-provider-spotinst: docs/resources/ocean_aks.md

* `name` - (Required) The Ocean cluster name.
* `controller_cluster_id` - (Required) A unique identifier used for connecting the Ocean SaaS platform and the Kubernetes cluster. Typically, the cluster name is used as its identifier. 
* `aks_name` - (Required) The AKS cluster name.

> * `acd_identifier` - (Required) The AKS identifier. Replace the <ACD_ID> with any unique string. The <ACD_ID> should be in the following format: “acd-XXXXXXXX” (you cannot use an ACD_ID that was previously used).

Reviewer: `<any unique string> != <specific format>` (contradiction).

```suggestion
* `acd_identifier` - (Required) The AKS identifier. A valid identifier should be formatted as `acd-nnnnnnnn` and previously used identifiers cannot be reused.
```

Author: done
